prow/config.yaml

```diff
@@ -38,6 +39,7 @@ tide:
     - needs-ok-to-test
     - do-not-merge/work-in-progress
     - do-not-merge/hold
+    - do-not-merge/blocked-paths
```

Comment:
should we just add all the `do-not-merge/*` labels in here?

https://github.com/kubernetes/test-infra/blob/661a6b261bcb2004ddede6b935b13629e1a8bf58/label_sync/labels.yaml#L29-L40

Reply:
Fair point.  I'd personally like to get rid of the unprefixed `do-not-merge` label by migrating it to one of the other labels, but that can be done orthogonally.  I'll add it here for now.